02-python-1/Python-1-Instructor-notes.ipynb

```diff
@@ -0,0 +1,378 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Comments\n",
+    "As a side, commenting out uses `#` and you can block comment a la"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "I am a leaf on the wind\n",
+    "Watch me soar\n",
+    "\"\"\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##Lists\n",
+    "Lists are an immensely powerful part of the Python toolbox. \n",
+    "A list can be of anything, strings, float, complex etc. Lists are iterable objects. Donted by square brackets, empty lists can be denoted by alist = []."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "alist = [1,2,3,4,5]\n",
+    "print alist"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then we can use append to stick some more stuff on the end. And we can find the first instance of something with index."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "alist.append(6)\n",
+    "print alist\n",
+    "print alist.index(6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also sort our lists using sort()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "anotherlist = ([6,2,5,2,7,1,3,5,4])\n",
+    "anotherlist.sort()\n",
+    "print anotherlist"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also make lists of lists, extremely useful when we start processing and holding lots of data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "lista = ([2,5,3])\n",
+    "listb = ([5,2,9])\n",
+    "listc = ([4,6,1])\n",
+    "list_total = (lista, listb, listc)\n",
+    "print list_total"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "###Dictionaries\n",
+    "Finally we also have dictionaries. Dictionaries are an unordered set of key and value pairs. An empty dictionary is denoted by adict = {}. We can then extract information from it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "times = {'Jones':27, 'Ames':26, 'Andrews':29}\n",
+    "print times.keys()\n",
+    "print times"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the keys are strings then you can use the dict method `dict = (some stuff)`"
```

Comment:
it is kinda useful, but you haven't introduced the type constructors i.e. `list()` anywhere else. Other opinions?